File: app/src/main/java/cn/vove7/accessibilityservicedemo/model/AppInfo.java
package cn.vove7.accessibilityservicedemo.model;

import android.graphics.drawable.Drawable;

public class AppInfo {
   String name;
   String packageName;
   Drawable icon;
   int pid;

   public String getName() {
      return name;
   }

   public void setName(String name) {
      this.name = name;
   }

   public String getPackageName() {
      return packageName;
   }

   public void setPackageName(String packageName) {
      this.packageName = packageName;
   }

   public Drawable getIcon() {
      return icon;
   }

   public void setIcon(Drawable icon) {
      this.icon = icon;
   }

   public int getPid() {
      return pid;
   }

   public void setPid(int pid) {
      this.pid = pid;
   }
}

File: vtp/src/main/java/cn/vove7/vtp/app/AppInfo.kt
package cn.vove7.vtp.app

import android.graphics.drawable.Drawable

/**
 * App信息类
 */
open class AppInfo(
        var name: String = "",
        var alias: String = name,
        var packageName: String = "",
        var icon: Drawable? = null,
        var pid: Int = 0,
        var priority: Int = 0
) {
    override fun toString(): String {
        return name
    }
}